docs/docs/troubleshooting.md

```diff
@@ -51,27 +51,44 @@ If you are reviewing these errors in the GitHub Actions tab, the contents of `<b
 
 ### Split Keyboard Halves Unable to Pair
 
-Previously, pairing split keyboard halves involved a **BLE Reset** via a combination of held keys that removed all bluetooth profile information from the keyboard.
-Since then, a much simpler procedure of performing a bluetooth reset for split keyboards has come about, without the need for any file modification:
+Split keyboard halves pairing issue can be resolved by flashing a settings reset firmware to both controllers. You will first need to acquire the reset UF2 image file with one of the following options:
 
-**New Procedure:**
+#### Option 1: Build Reset UF2 in 'zmk-config'
+
+Find the `build.yaml` file in your `zmk-config` folder and add an additional settings reset build for the board used by your split keyboard. For example assume that we have setup nice!nano v2 for Corne, apppend the following `settings_reset` shield to the `build.yaml` file:
+
+```yml
+include:
+  - board: nice_nano_v2
+    shield: corne_left
+  - board: nice_nano_v2
+    shield: corne_right
+  - board: nice_nano_v2
+    shield: settings_reset
+```
+
+Save the file, commit the changes and push them to GitHub. Download the new firmware zip file build by the latest GitHub Actions job. In it you will find an additional `settings_reset` UF2 image file.
+
+#### Option 2: Download Reset UF2 from ZMK's Workflow
 
 1. [Open the GitHub `Actions` tab and select the `Build` workflow](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain+event%3Apush).
-1. Find one of the 'results' for which the core-coverage job was successfully run, indicated by a green checkmark in the core-coverage bubble.
-1. From the next page under "Artifacts", download the `$boardname-settings_reset-zmk` zip file.
-1. Unzip the downloaded file.
-1. Put each half of the split keyboard into bootloader mode
-1. Flash one of the halves of the split with the "settings clear" UF2 image from step 1. Immediately after flashing "settings clear" to the chosen half, immediately put it into bootloader mode
-   to avoid accidental bonding between the halves.
-1. Repeat step 3 with the other half of the split keyboard
-1. Flash the actual image for each half of the split keyboard (e.g `my_board_left.uf2` to the left half, `my_board_right.uf2` to the right half)
+2. Find one of the 'results' for which the core-coverage job was successfully run, indicated by a green checkmark in the core-coverage bubble like the image example below.
+3. From the next page under "Artifacts", download and unzip the `$boardname-settings_reset-zmk` zip file for the UF2 image.
```

Comment:
```suggestion
3. From the next page under "Artifacts", download and unzip the `<board_name>-settings_reset-zmk` zip file for the UF2 image.
```
What do you think about this change while we are at it? I think this templating is a bit clearer and the previous one caused some confusion with users, but at the end of the day it is subjective.

Reply:
I agree with this suggestion for consistency.